docs/book/src/plugins/grafana-v1-alpha.md

```diff
@@ -0,0 +1,96 @@
+# Grafana Plugin
+
+The Grafana plugin provides the Grafana manifest that you can use to visualize your operator status easily.
+
+<aside class="note">
+<h1>Examples</h1>
+
+Samples are provided under the [testdata][testdata] directory of the Kubebuilder project.
+You can check samples using this plugin by looking at the `project-v3-with-grafana` projects under the [testdata][testdata] directory on the root directory of the Kubebuilder project.
+
+</aside>
+
+## When to use it ?
+
+- If you want to observe your operator runtime status based on the [controller metrics][controller-metrics] collected by Prometheus
+
+## How to use it ?
+
+### Prerequisites:
+
+1. Prometheus monitoring stacks running in your K8s cluster (e.g: [Prometheus Operator][prom-operator], [kube-prometheus][kube-prometheus])
+2. A servicemonitor that exposes your operator metrics to Prometheus
+   - If you're using the kustomize plugin in the meantime, you should have the manifest under `config/prometheus`
+3. Users must have write access to their grafana services
+
+### Sample of servicemonitor
+
+```yaml
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: jaeger-operator-servicemonitor
+  namespace: observability
+  labels:
+    app: jaeger-operator
+    release: prometheus
+spec:
+  selector:
+    matchLabels:
+      name: jaeger-operator # target service
+  endpoints:
+    - path: /metrics
+      port: https
+      scheme: https
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      tlsConfig:
+        insecureSkipVerify: true
+```
+
+### Basic Usage
+
+The Grafana plugin is attached to the `init` subcommand:
+
+```sh
+kubebuilder init --plugins grafana.kubebuilder.io/v1-alpha
+```
+
+Once triggered, there will be a `grafana/controller-overall.json` file scaffolded out.
+This json file utilizes [controller-runtime][controller-runtime] default metrics to display your operator runtime status.
+
+Bascially, you can:
+
+1. Copy this JSON file
+2. Create a new dashboard in your Grafana Web UI
+3. In the setting page of the dashboard, paste the content to its JSON model
+4. Save the model, and there should be a newly created dashboard `Kubebuiler-Grafana-Plugin`
+
+## Subcommands
+
+The Grafana plugin implements the following subcommands:
+
+- init (`$ kubebuilder init [OPTIONS]`)
```

Comment:
```suggestion
- edit (`$ kubebuilder init [OPTIONS]`)
- init (`$ kubebuilder init [OPTIONS]`)
```

Reply:
Could we add the edit? TO do so we just need to add a new edit.go in the plugin that does exactly the same scaffold and steps done by the init.